pymc3/tests/test_distributions.py

```diff
@@ -806,16 +806,28 @@ def test_fun(value, mu, alpha):
         "mu, p, alpha, n, expected",
         [
             (5, None, None, None, "Incompatible parametrization. Must specify either alpha or n."),
-            (None, .5, None, None, "Incompatible parametrization. Must specify either alpha or n."),
-            (None, None, None, None, "Incompatible parametrization. Must specify either alpha or n."),
+            (
+                None,
+                0.5,
+                None,
+                None,
+                "Incompatible parametrization. Must specify either alpha or n.",
+            ),
+            (
+                None,
+                None,
+                None,
+                None,
+                "Incompatible parametrization. Must specify either alpha or n.",
+            ),
```

Comment:
Can you change L834 so that it reads
```python
with pytest.raises(ValueError, match=f"Incompatible parametrization. {expected}"):
```
and then `"Incompatible parametrization."` can be removed from here (and from the remaining tuples from this particular test):
```python
(None, None, None, None, "Must specify either alpha or n."),
```
 and `black` will keep this on one line.

You'll then need to run `pytest pymc3/tests/test_distributions.py -k test_negative_binomial_init_fail` to check that this still passes

If you decide to do this, please comment in the PR description that you've made this change. In general it's good to keep formatting separate from changing code (this way it's easier to review / revert), but IMO here it's a really simple change and it allows the formatting to be applied more cleanly so it's OK